src/index.js

```diff
@@ -26,53 +26,13 @@ import {
   ExtraLarge,
   LargerThanExtraSmall,
 } from './Responsive';
-import SearchField from './SearchField';
-import StatefulButton from './StatefulButton';
-import StatusAlert from './StatusAlert';
-import Table from './Table';
-import Tabs from './Tabs';
-import TextArea from './TextArea';
-import ValidationFormGroup from './ValidationFormGroup';
-import TransitionReplace from './TransitionReplace';
-import ValidationMessage from './ValidationMessage';
-import Variant from './utils/constants';
-
-export {
```

Comment:
Getting rid of this big, single export is also very important.  You can't tree shake a single export, they need to be individual exports.